storagenode/orders/db_test.go

```diff
@@ -46,7 +47,9 @@ func TestOrders(t *testing.T) {
 		require.NoError(t, err)
 		require.Len(t, emptyArchive, 0)
 
-		now := ptypes.TimestampNow()
+		now := time.Now().UTC()
+		nowTimestamp, err := ptypes.TimestampProto(time.Now().UTC())
```

Comment:
How is this different than `ptypes.TimestampNow()`?

Reply:
oh oops that's a typo they were suppose to be created from the same time

Reply:
`nowTimestamp, err := ptypes.TimestampProto(now)`